alf/config_util.py

```diff
@@ -250,6 +262,28 @@ def config1(config_name, value, mutable=True):
         config_node.set_mutable(mutable)
 
 
+def get_config(config_name):
```

Comment:
Maybe we can change the name of this function to ```get_config_value```?

Reply:
changed